src/Appium.Net/Appium/Service/AppiumLocalService.cs

```diff
@@ -65,24 +61,14 @@ internal AppiumLocalService(
             Port = port;
             InitializationTimeout = initializationTimeout;
             EnvironmentForProcess = environmentForProcess;
-#if !NET48
-            SharedHttpClient = CreateHttpClientInstance;
-#endif
+            SharedHttpClient = CreateHttpClientInstance();
         }
 
-#if !NET48
-        private HttpClient CreateHttpClientInstance
```

Comment:
@nvborisenko have you run some tests against .NET framework 4.7+?

Reply:
Do you know any known issues?

Reply:
Only .net framework 4.7 is under interest? `netstandard2.0` is supported by .net framework 4.6.1

Reply:
You said you were trying to run with. Net 4.7, so this is why I asked. 
:)

Reply:
> Do you know any known issues?

Should be the same issues I had for 4.8 😁

Reply:
Really, what issues?

Reply:
Basically when I reintroduced 4.8.
https://github.com/appium/dotnet-client/pull/740
Maybe things have changed since then.
I can have a look later if you unable to run the tests

Reply:
LOL, now I see:

![image](https://github.com/user-attachments/assets/da8ee502-fac7-48d5-905b-a01d036d0e5d)

Reply:
Just believe me, this PR is about "Use netstandard2.0 as target framework to support **wider frameworks**"

Reply:
I am not from Selenium team, I am contributing as personal.

Reply:
As for Selenium:

![image](https://github.com/user-attachments/assets/51bd38ad-34ac-42ba-98d3-eebbb5d3ca7e)

Reply:
Ok. Let's go ahead with this change.
Fingers crossed :)

Reply:
Selenium is already felt it.